### ISSUE TITLE
Icons disappear after restart

### ISSUE THREAD

**Issue by Devenda** [FlorianWoelki/obsidian-icon-folder#163]
**Describe the bug**
Icons disappear after restart.
It seems to have started after updating Obsidian to version 1.2.0.
Reinstall of the plugin did not work.

*Warning*: reinstalling the plugin removes all config, so make sure to keep a backup of the previous config (data.json in the plugin folder).

*Edit*: I do not have sync enabled, only Git is active for syncing to GitHub.

**To Reproduce**
1. Add an Icon to a file or folder
2. Check the data.json file in the plugin folder to verify that the changes are saved
3. Close Obsidian
4. Start Obsidian
5. Icon does not appear
6. Verify data.json to see that the config is still present

**Expected behavior**
Previous icons to be displayed

**Additional context**
The issue seems to have started when updating Obsidian, after the restart the plugin did no longer show in the overview under 'Community plugins'. After reinstalling the plugin the plugin started to work, however icons are no longer shown.

**Comment by Devenda** [comment]
The error in the console:
![image](https://user-images.githubusercontent.com/2527506/228788649-e4d658cb-5f18-41b9-8b5e-c0fedfd701c8.png)

**Comment by NightOwlCoder** [comment]
Here is my error, looks the same as @Devenda 
```
Uncaught TypeError: Cannot read properties of undefined (reading 'children')
    at eval (plugin:obsidian-icon-folder:2283:29)
    at Array.forEach (<anonymous>)
    at eval (plugin:obsidian-icon-folder:2277:14)
    at Array.forEach (<anonymous>)
    at addIconsToDOM (plugin:obsidian-icon-folder:2267:19)
    at IconFolderPlugin.handleChangeLayout (plugin:obsidian-icon-folder:3892:9)
    at eval (plugin:obsidian-icon-folder:3797:82)
    at t.tryTrigger (app.js:1:939830)
    at t.trigger (app.js:1:939763)
    at e.trigger (app.js:1:1867928)
eval @ plugin:obsidian-icon-folder:2283
eval @ plugin:obsidian-icon-folder:2277
addIconsToDOM @ plugin:obsidian-icon-folder:2267
handleChangeLayout @ plugin:obsidian-icon-folder:3892
eval @ plugin:obsidian-icon-folder:3797
t.tryTrigger @ app.js:1
t.trigger @ app.js:1
e.trigger @ app.js:1
e.updateLayout @ app.js:1
r @ app.js:1
```

**Comment by NightOwlCoder** [comment]
I fixed that error by checking for null, but icons still do not load.
```
if (titleEl && titleEl.children && (titleEl.children.length === 2 || titleEl.children.length === 1)) {
```

**Comment by FlorianWoelki** [comment]
This is fixed with #165, but I cannot test nor review it because I cannot access the insider builds. I'm very sorry about that. I'll fix this whenever v2.0.0 is publicly available.

**Comment by Devenda** [comment]
Great! Thank you for the heads up and for maintaining the plugin!
